docs/manage/_command_line.rst

```diff
@@ -0,0 +1,184 @@
+.. _command_line:
+
+
+Working with Kolibri from the Command Line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Are there any risks to publicly documenting some of these commands? My understanding was that these were internal and not external.

We should be careful to only document the stuff we're ready to support, and move anything else to developer docs.

Documenting some of this stuff (e.g. environment variables) locks us into de facto longer-term support for features and configuration that we haven't committed to.

Reply:
I understand the concern, but even env vars are not set in stone for all the eternity, right...?
 
In KA Lite docs we had more and more complex command line instructions offered to advanced users. As for them being "internal", we have shared them with many partners and users while offering support, that the cat is definitely out of the bag :scream_cat: 

Maybe add an alert in spirit of `For advanced users only!`...?

Reply:
We share it with partners, but we also have a lot more bandwidth to support them if/when things go wrong.

I'm fine with either way; just wanted to flag it as a concern.

Reply:
@indirectlylit good concern -- I looked through the mentioned commands, and I think `Import Content Channels from a Local Drive` has a doubtful API, so removing that!